docker/Dockerfile

```diff
@@ -0,0 +1,13 @@
+FROM ghcr.io/puppeteer/puppeteer:17.0.0
+# LABEL org.opencontainers.image.authors="Mogztter#Guillaume Grossetie"
+LABEL version="17.0.0"
+LABEL description="This docker image contains \
+asciidoctor-web-pdf to generate websites and pdf from asciidoc(tor)."
+
+ENV PUPPETEER_SKIP_DOWNLOAD=true \
+    PUPPETEER_EXECUTABLE_PATH=/usr/bin/google-chrome
+
+USER root
+WORKDIR /usr/src/app
+
+RUN npm install -g @asciidoctor/core asciidoctor-pdf --save-dev
```

Comment:
I think it would be better to copy local files and run `npm i` + `npx asciidoctor-web-pdf` (or `node ./bin/asciidoctor-web-pdf`).

We should also set `NODE_ENV="production"` to avoid installing development dependencies.

Reply:
Adding the environment is a good idea. Agree.

I am not very experienced with npm (apart from using it). Do I understand you correctly to download `asciidoctor-web-pdf` and install it?

For instance like this [puppeteer Dockerfile](https://github.com/puppeteer/puppeteer/blob/main/docker/Dockerfile#L17-L20)

```Dockerfile
COPY puppeteer-latest.tgz /home/pptruser/puppeteer-latest.tgz

# Install puppeteer into /home/pptruser/node_modules.
RUN npm i ./puppeteer-latest.tgz \
...
```

Reply:
Please check the latest Dockerfile.